tests/test_quick_start.py

```diff
@@ -143,7 +143,8 @@
     },
     "squad": {
         ("squad/squad.json", "squad_model", ALL_MODES): [TWO_ARGUMENTS_INFER_CHECK],
-        ("squad/squad_ru.json", "squad_model_ru", ALL_MODES): [TWO_ARGUMENTS_INFER_CHECK]
+        ("squad/squad_ru.json", "squad_model_ru", ALL_MODES): [TWO_ARGUMENTS_INFER_CHECK],
+        ("squad/multi_squad_noans.json", "multi_squad_noans", ALL_MODES): [TWO_ARGUMENTS_INFER_CHECK]
```

Comment:
Test `test_consecutive_training_and_interacting` for "multi_squad_noans" takes to much time (I had to terminate it). Check test parameters.